src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -150,6 +150,22 @@ public class CodeStyleOptions
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("dotnet_style_prefer_auto_properties"),
                 new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferAutoProperties") });
 
+        internal static readonly PerLanguageOption<CodeStyleOption<bool>> PreferInferredTupleNames = new PerLanguageOption<CodeStyleOption<bool>>(
+            nameof(CodeStyleOptions),
+            nameof(PreferInferredTupleNames),
+            defaultValue: TrueWithSuggestionEnforcement,
+            storageLocations: new OptionStorageLocation[] {
+                EditorConfigStorageLocation.ForBoolCodeStyleOption("dotnet_style_prefer_inferred_tuple_names"),
```

Comment:
Might be worth calling out somewhere this is an explicit back compat break.

Reply:
@jinujoseph @kuhlenh  Is there a place to document code compat breaks in code style options?

Reply:
I can add a note to the documentation that it used to be separate but now it is not (and that the previous options are not supported). I never documented the previous version, so we should be ok...